RevenueCatUI/CustomerCenter/Views/CompatibilityContentUnavailableView.swift

```diff
@@ -30,12 +30,20 @@ struct CompatibilityContentUnavailableView: View {
 
     var body: some View {
 
-        if #available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *) {
-            ContentUnavailableView(
-                title,
-                systemImage: systemImage,
-                description: Text(description)
-            )
+        if #available(iOS 17.0, *) {
+            #if swift(>=5.9)
```

Comment:
this is an old trick that's useful for these Xcode transitions - you point to the min swift version that's packed with the newest xcode, and it effectively works as "if xcode is greater than 15"

Reply:
Cool!